tests/shipping-preview/models/Delivery Only_Delivery Pickup.model.js

```diff
@@ -0,0 +1,45 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  checkShippingPreviewResult,
+  choosePickupShippingPreview,
+  fillShippingPreviewDelivery,
+} from '../../../utils/shipping-actions'
+import { ACCOUNT_NAMES, SKUS, SLA_IDS } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Delivery Only + Delivery/Pickup - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only pickup', () => {
+      setup({
+        skus: [
+          SKUS.PICKUP_1_SLA_AND_DELIVERY_MULTIPLE_SLA,
+          SKUS.DELIVERY_MULTIPLE_SLA_AND_PICKUP_AT_PORTO_ALEGRE,
+        ],
+        account,
+      })
+
+      const selectors = []
+
+      cy.contains('Calcular').should('be.visible')
```

Comment:
Why did you choose to check by `Calcular` text instead of `shipping-calculate-link` id?

Reply:
This check wasn't meant to this. Actually, it is unnecessary since the subsequent actions are gonna look for the `shipping-calculate-link` id. So, I think I should remove it from the models.

Reply:
I just removed this check from the models. So I'm asking your review again.